bugbug/test_scheduling.py

```diff
@@ -117,6 +117,12 @@
 )
 
 
+class UnexpectedGranularityError(Exception):
```

Comment:
This may be better to inherate from `ValueError`

Reply:
class UnexpectedGranularityError(ValueError):
    def __init__(self, granularity):
        message = f"Unexpected {granularity} granularity"
        super().__init__(message)

Reply:
👍